### PR TITLE
fix(admin-ui): Collapse categories by default

### PR DESCRIPTION
**Pull request by olivermrbl**
Expanded by default makes the overview a bit chaotic at first glance:

CleanShot 2023-03-30 at 11 35 38@2x

### CODE REVIEW

**Review by adrien2p — APPROVED** [review]
LGTM, thought, should we add a searchable field?

File: packages/admin-ui/ui/src/domain/product-categories/components/product-categories-list.tsx [hunk 1]
```diff
@@ -76,6 +76,7 @@ function ProductCategoriesList(props: ProductCategoriesListProps) {
     () => (
       <Nestable
         items={categories}
+        collapsed={true}
```
Comment (fPolic) [inline packages/admin-ui/ui/src/domain/product-categories/components/product-categories-list.tsx:79]:
```suggestion
        collapsed
```

**Review by fPolic — APPROVED** [review]
LGTM!

**Comment by olivermrbl** [comment]
> should we add a searchable field?

Yes, I think that would be an excellent addition. Already on @riqwan's radar, I believe.

**Comment by olivermrbl** [comment]
@riqwan Are you cool with this PR? Don't want to trump this through without your approval.

**Pull request merged**